Sources/MapboxMaps/Annotations/Generated/CircleAnnotation.swift

```diff
@@ -129,6 +135,33 @@ public struct CircleAnnotation: Annotation {
         }
     }
 
+
+    /// Get the offset geometry for the touch point
+    func getOffsetGeometry(_ mapboxMap: MapboxMap, moveDistancesObject: MoveDistancesObject?) -> Geometry? {
+        let maxMercatorLatitude = 85.05112877980659
+        let minMercatorLatitude = -85.05112877980659
+
+        guard let moveDistancesObject = moveDistancesObject else {
+          return nil
+        }
+
+           let point = point.coordinates
+
+        let centerScreenCoordinate = mapboxMap.point(for: point)
+
+        let targetCoordinates =  mapboxMap.coordinate(for: CGPoint(x: centerScreenCoordinate.x - moveDistancesObject.distanceXSinceLast, y: centerScreenCoordinate.y - moveDistancesObject.distanceYSinceLast))
+
+        let targetPoint = Point(targetCoordinates)
+
+        let shiftMercatorCoordinate = Projection.calculateMercatorCoordinateShift(startPoint: Point(point), endPoint: targetPoint, zoomLevel: mapboxMap.cameraState.zoom)
+
+        let targetPoints = Projection.shiftPointWithMercatorCoordinate(point: Point(point), shiftMercatorCoordinate: shiftMercatorCoordinate, zoomLevel: mapboxMap.cameraState.zoom)
+
+        if targetPoints.coordinates.latitude > maxMercatorLatitude || targetPoints.coordinates.latitude < minMercatorLatitude {
+           return nil
+        }
```

Comment:
Please, always prefer `guard` statement over `if` that follows by `return`/`throw` in condition branch
```suggestion
        guard validMercatorLatitude.contains(targetPoints.coordinates.latitude) else {
            return nil
        }
```